.github/workflows/go.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           submodules: recursive
 
       - name: Set up Go
-        uses: actions/setup-go@v5.4.0
+        uses: actions/setup-go@v5.3.0
```

Comment:
Why downgrade here?

Reply:
ah this should not have been changed, that was a bad merge from main